internal/e2e/lds_test.go

```diff
@@ -86,7 +87,7 @@ func TestNonTLSListener(t *testing.T) {
 			&v2.Listener{
 				Name:         "ingress_http",
 				Address:      envoy.SocketAddress("0.0.0.0", 8080),
-				FilterChains: envoy.FilterChains(envoy.HTTPConnectionManager("ingress_http", envoy.FileAccessLogEnvoy("/dev/stdout"))),
+				FilterChains: envoy.FilterChains(envoy.HTTPConnectionManager("ingress_http", envoy.FileAccessLogEnvoy("/dev/stdout"), ptypes.DurationProto(0))),
```

Comment:
I've put the zero value as simply as I could, not sure if that or having a helper of some sort is better.

Reply:
If we can define the type of duration as time.Duration, then you can just write
```
FilterChains: envoy.FilterChains(envoy.HTTPConnectionManager("ingress_http", envoy.FileAccessLogEnvoy("/dev/stdout"),0))
```